### PR TITLE
feat: new cask EasyDevo

### PR DESCRIPTION
**Pull request by frostming**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by STRRL** [comment]
Awesome! Thanks a lot @frostming :heart:

> /cc @STRRL PTAL in case there are more application files not listed in the `zap trash` field.

I think no more items should be included in zap trash, thx!

File: Casks/e/easydevo.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  url "https://github.com/boring-design/EasyDevo-Distro/releases/download/v#{version}/EasyDevo-Mac-#{version}-Installer.dmg",
+      verified: "github.com/boring-design/EasyDevo-Distro/"
+  name "EasyDevo"
+  desc "Elegant tool built for coding"
+  homepage "https://easydevo.boringboring.design/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (SMillerDev) [inline Casks/e/easydevo.rb:14]: Will there be pre-releases in this repo?

Comment (frostming) [reply Casks/e/easydevo.rb:14]: @STRRL , if so, how should we update the strategy?

Comment (SMillerDev) [reply Casks/e/easydevo.rb:14]: This trategy is only for pre-releases. Otherwise the livecheck can be omitted and use the default Git strategy

Comment (frostming) [reply Casks/e/easydevo.rb]: Thanks for your prompt response, I've removed the livecheck block.

**Review by p-linnane — APPROVED** [review]
Thanks for contributing to Homebrew @frostming!

**Pull request merged**